manual_build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+GORELEASER_IMAGE=iotmod/goreleaser-cgo-cross-compiler:1.13.5
+REPO_PATH="/build"
+PWD="$(pwd)"
+
+mkdir -p ${PWD}
+docker pull ${GORELEASER_IMAGE}
+docker run --rm --privileged -v $PWD:$REPO_PATH -w $REPO_PATH ${GORELEASER_IMAGE} goreleaser --release-notes=CHANGELOG.md --rm-dist --snapshot
```

Comment:
I guess this script is just there to check, if GoReleaser is working correctly?

* We should not publish such a manual build
* The changelog flag is outdated since we don't publish a changelog automatically anymore

```suggestion
docker run --rm --privileged -v $PWD:$REPO_PATH -w $REPO_PATH ${GORELEASER_IMAGE} goreleaser --rm-dist --snapshot --skip-publish
```